fake_switches/switch_configuration.py

```diff
@@ -175,6 +175,7 @@ def reset(self):
         self.lldp_med_transmit_network_policy = None
         self.spanning_tree = None
         self.spanning_tree_portfast = None
+        self.ntp_disabled = None
```

Comment:
wouldn't it be better to have "ntp = True/False/None" ?  instead of "disabled" in the title?

Reply:
ok